gensim/models/ldaseqmodel.py

```diff
@@ -371,6 +371,53 @@ def doc_topics(self, doc_number):
         return doc_topic[doc_number]
 
 
+    def DTMvis(self, time, corpus):
+        """
+        returns term_frequency, vocab, doc_lengths, topic-term distributions and doc_topic distributions, specified by pyLDAvis format.
+        all of these are needed to visualise topics for DTM for a particular time-slice via pyLDAvis.
+        input parameter is the year to do the visualisation.
+        """
+
+        doc_topic = numpy.copy(self.gammas)
+        doc_topic /= doc_topic.sum(axis=1)[:, numpy.newaxis]
```

Comment:
Why not simply `self.gammas / self.gammas.sum(axis=1)[:, numpy.newaxis]`? Are you running out of memory?

If so, put a clear comment to that effect here, so somebody doesn't accidentally "fix" the code later.